### PR TITLE
AO3-4658 Log in page needs mobile styling

### PR DESCRIPTION
**Pull request by sarken**
https://otwarchive.atlassian.net/browse/AO3-4658
## Purpose

Updates the layout used for ao3.org/login to make it match the rest of the site and therefore actually acquire the mobile CSS. Also hides the giant honking image on narrow screens so the content can take up the full width, much like #2493 does with error pages

### CODE REVIEW

File: app/views/layouts/session.html.erb [hunk 1]
```diff
@@ -14,26 +14,28 @@
...
-      <%= flash_div :error, :caution, :notice %>
-      <p id="logo">
-        <%= image_tag(ArchiveConfig.LOGO, :alt => ArchiveConfig.ALT_LOGO) %>
-      </p>
-      <div id="signin" role="main">
-        <%= yield %>
+  <body class="<%= logged_in? ? 'logged-in' : 'logged-out' %>" <%= PhraseApp::InContextEditor.enabled? ? "data-no-turbolink" : "" %>>
+    <div id="outer" class="wrapper">
+      <ul id="skiplinks"><li><a href="#main"><%= ts('Main Content') %></a></li></ul>
+      <%= render :partial => 'layouts/header' %>
```
Comment (ariana-paris) [inline app/views/layouts/session.html.erb]: Could this also be 
```
<% render 'layouts/header' %>
```

Comment (sarken) [inline app/views/layouts/session.html.erb]: Yep! Just missed that one.

Comment (sarken) [inline app/views/layouts/session.html.erb]: Fixed now -- gotta love GitHub's editor :P

**Comment by sarken** [comment]
I should have ignored you and force-pushed, gah.

**Pull request closed without merging**